docs/content/preview/yugabyte-platform/install-yugabyte-platform/install/kubernetes.md

```diff
@@ -37,7 +37,7 @@ For installation overview, select one of the following installation types:
   </li>
 </ul>
 
-<br>Click elements of the following chart to access detailed steps:
+<br>The following diagram depicts the YugabyteDB Anywhere installation process in the Kubernetes environment:
```

Comment:
Think the spacing below tabs was fixed, br shouldn't be needed

Reply:
deleted